src/tests/ftest/security/pool_security_acl.py

```diff
@@ -0,0 +1,421 @@
+        permission, read, write = self.params.get("name", "/run/pool_acl/*")
+
+        user_types  = ["owner", "user", "ownergroup", "group", "everyone"]
+        default_acl_entries = ["A::OWNER@:",
+                               self.acl_entry("user", self.current_user, ""),
+                               "A:G:GROUP@:",
+                               self.acl_entry("group", self.current_group, ""),
+                               "A::EVERYONE@:"]
+        test_acl_entries   = ["", "", "", "", ""]
+
+        if permission.lower() == "none":
+            permission = ""
+        if permission not in PERMISSIONS:
+            self.fail("##permission {} is invalid, valid permissions are:"
+                      "'none', 'r', w', 'rw'".format(permission))
+        if user_type not in user_types:
+            self.fail("##user_type {} is invalid, valid user_types are:"
+                      .format(user_type, user_types))
+        user_type_ind = user_types.index(user_type)
+        self.log.info("===>Start DAOS pool acl enforcement order Testcase: "
+                      " user_type: {}, permission: {}, expect_read: {},"
+                      " expect_write: {}."
+                     .format(user_type, permission, read, write))
+        #take care of the user_type which have higher priviledge
+        for ind in range(5):
+            if ind < user_type_ind:
+                test_acl_entries[ind] = ""
+            elif ind == user_type_ind:
+                continue
+            else:
+                test_acl_entries[ind] = default_acl_entries[ind]
+        test_permission = permission
+        #take care of rest of the user-type permission
+        group_acl = ""
+        for ind in range(user_type_ind, 5):
+            if ind != user_type_ind:
+                #setup opposite test_permission with permission
+                test_permission = "rw"
+                if permission == "rw":
+                    test_permission = ""
+                if user_types[ind] == "group":
+                    group_acl = test_permission
+            #"A::OWNER@:" + permission
+            test_acl_entries[ind] = default_acl_entries[ind] + test_permission
+        #union of ownergroup and group permission
+        if user_type == "ownergroup":
+            if permission != group_acl:
+                union_acl = "".join(list(set().union(permission,
+                    group_acl)))
+                if union_acl == "":
+                    read = "deny"
+                    write = "deny"
+                elif union_acl == "r":
+                    read = "pass"
+                    write = "deny"
+                elif union_acl == "w":
+                    read = "deny"
+                    write = "deny"
+                else:
+                    read = "pass"
+                    write = "pass"
+        self.pool_acl_verification(test_acl_entries, read, write)
+        self.log.info("--->Testcase Passed. "
```

Comment:
(pylint-logging-format-interpolation) Use % formatting in logging functions and pass the % parameters as arguments